pingouin/multivariate.py

```diff
@@ -254,3 +254,114 @@ def multivariate_ttest(X, Y=None, paired=False):
     stats = {'T2': t2, 'F': fval, 'df1': df1, 'df2': df2, 'pval': pval}
     stats = pd.DataFrame(stats, index=['hotelling'])
     return _postprocess_dataframe(stats)
+
+
+def box_m(data, dvs=None, group=None, alpha=.001):
+    """Test equality of covariance matrices.
+
+    Parameters
+    ----------
+    data : :py:class:`pandas.DataFrame`,
+    should be a long-format pandas dataframe.
+    dvs : str
+        Variables of covariance matrix (only when ``data``
+        is a long-format dataframe).
+    group : str
+        Grouping variable (only when ``data`` is a long-format dataframe).
+    alpha : float
+        Significance level.
+
+    Returns
+    -------
+    stats : :py:class:`pandas.DataFrame`
+
+        * ``'Chi2'``: Test statistic
+        * ``'pval'``: p-value
+        * ``'df'``: The Chi-Square statistic's degree of freedom
+        * ``'equal_cov'``: True if ``data`` has equal covariance
+
+    Notes
+    -----
+    This function does not handle missing values. Please
+    handle them (e.g. remove or impute these values)
+    prior to perform the Box's M test.
+    Pooled sample covariance matrix :math:`S_{\\text{pl}}`
+    is calculated as
+    .. math::
```

Comment:
Please add space before and after the math code block, otherwise it won't be rendered properly in the HTML documentation

Reply:
You mean there should be a blank line before and after the "..math::"? Is there a way that I can see how it renders locally?

Reply:
@GanshengT the instructions for building the documentation can be found here: https://pingouin-stats.org/contributing.html